app/js/command.js

```diff
@@ -452,6 +457,7 @@ const commands = {
     "quitall": quitall,
     "devtools": () => TABS.currentPage().openDevTools(),
     "reload": reload,
+    "restart": restart,
```

Comment:
Please add "restart" to the list of "noArgumentComands" just below. This is a list of commands that don't have any arguments, and it will throw a generic warning for these commands if they contain trailing arguments.

Reply:
Done.